packages/rpc-graphql/src/__tests__/account-test.ts

```diff
@@ -1,12 +1,13 @@
 import {
+    createRpc,
     createSolanaRpcApi,
     GetAccountInfoApi,
     GetBlockApi,
     GetProgramAccountsApi,
     GetTransactionApi,
-} from '@solana/rpc-core';
-import { createHttpTransport, createJsonRpc } from '@solana/rpc-transport';
-import type { Rpc } from '@solana/rpc-types';
+    Rpc,
+} from '@solana/rpc';
+import { createHttpTransport } from '@solana/rpc-transport-http';
```

Comment:
Up to you, but I think `createDefaultHttpTransport` from `@solana/rpc` would work here, and then the transport lib isn't needed anywhere. The rest of the library should only need `@solana/rpc`.

Reply:
That's a good shout, I'll update accordingly.